website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -731,6 +731,29 @@ alias: bottlerocket@v1.20.4
 ```
 The Windows family does not support pinning, so only `latest` is supported.
 
+The following commands can be used to determine the versions availble for an alias in your region:
+
+{{< tabpane text=true right=false >}}
+  {{% tab "AL2023" %}}
+  ```bash
+  export K8S_VERSION=1.30
```

Comment:
Can we reference this as `"{{< param "latest_release_version" >}}"` so that this gets auto-updated whenever we increase this version in the future?